jota/src/main/java/jota/IotaAPI.java

```diff
@@ -574,13 +561,12 @@ public GetBalancesAndFormatResponse getInputs(String seed, int security, int sta
      * @param addresses The addresses.
      * @param threshold Min balance required.
      * @param start     Starting key index.
-     * @param end       Ending key index.
      * @param stopWatch the stopwatch.
      * @param security  The security level of private key / seed.
      * @return Inputs object.
      * @throws InvalidSecurityLevelException is thrown when the specified security level is not valid.
      **/
-    public GetBalancesAndFormatResponse getBalanceAndFormat(final List<String> addresses, long threshold, int start, int end, StopWatch stopWatch, int security) throws InvalidSecurityLevelException {
+    public GetBalancesAndFormatResponse getBalanceAndFormat(final List<String> addresses, long threshold, int start, StopWatch stopWatch, int security) throws InvalidSecurityLevelException {
```

Comment:
Could you make this take a pair of `(String, int)` (i.e. `(address, keyIndex)`) instead of taking a start index and incrementing by one?
The current approach requires addresses to be ordered & have no missing key indices in between.